internal/services/hdinsight/hdinsight_cluster_pool_resource.go

```diff
@@ -130,9 +132,10 @@ func (r ClusterPoolResource) Arguments() map[string]*pluginsdk.Schema {
 		"log_analytics_profile": {
 			Type:     pluginsdk.TypeList,
 			Optional: true,
+			MaxItems: 1,
 			Elem: &pluginsdk.Resource{
 				Schema: map[string]*pluginsdk.Schema{
-					"log_analytics_profile_enabled": {
+					"enabled": {
```

Comment:
Can we remove the `enabled` field completely to conform to the new [Hashi 4.0 coding standards](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/schema-design-considerations.md#features-that-are-toggled-by-the-property-enabled)? So in this case if the `log_analytics_profile` block is in the config the profile would be `enabled` and if the `log_analytics_profile` is not in the config it would be `disabled`.

Reply:
Sure, will change.